spec/js/views/todoView_spec.js

```diff
@@ -62,28 +51,27 @@ define([
   });
 
   describe('delete', function () {
-    var todoListView = new TodoListView({collection: new TodoList(), el: $('<ul></ul>')}).render();
+    var todoList = new TodoList();
```

Comment:
As discussed before, never setup mutable objects outside a beforeEach / it function. This will be the same instance during the execution of all the specs in this describe